.palantir/revapi.yml

```diff
@@ -1146,6 +1146,10 @@ acceptedBreaks:
         \ org.apache.iceberg.TableMetadata)"
       justification: "Removing deprecated code"
   "1.7.0":
+    org.apache.iceberg:iceberg-api:
```

Comment:
we can't break the API. I commented in https://github.com/apache/iceberg/pull/12010/files#r1922279411 on how to avoid this, so the changes here can be removed

Reply:
Thanks @nastra - fixed in the latest commit